### ISSUE TITLE
2024-04-23-10

### ISSUE THREAD

**Issue by Mr-wang0129** [Mr-wang0129/ChatDailyPapers#2]
# image matching
## Scene Coordinate Reconstruction: Posing of Image Collections via Incremental Learning of a Relocalizer
- **Url**: http://arxiv.org/abs/2404.14351v1
- **Authors**: ['Eric Brachmann', 'Jamie Wynn', 'Shuai Chen', 'Tommaso Cavallari', 'Áron Monszpart', 'Daniyar Turmukhambetov', 'Victor Adrian Prisacariu']
- **Abstrat**: We address the task of estimating camera parameters from a set of images depicting a scene. Popular feature-based structure-from-motion (SfM) tools solve this task by incremental reconstruction: they repeat triangulation of sparse 3D points and registration of more camera views to the sparse point cloud. We re-interpret incremental structure-from-motion as an iterated application and refinement of a visual relocalizer, that is, of a method that registers new views to the current state of the reconstruction. This perspective allows us to investigate alternative visual relocalizers that are not rooted in local feature matching. We show that scene coordinate regression, a learning-based relocalization approach, allows us to build implicit, neural scene representations from unposed images. Different from other learning-based reconstruction methods, we do not require pose priors nor sequential inputs, and we optimize efficiently over thousands of images. Our method, ACE0 (ACE Zero), estimates camera poses to an accuracy comparable to feature-based SfM, as demonstrated by novel view synthesis. Project page: https://nianticlabs.github.io/acezero/

1. Title: 场景坐标重建：通过增量学习重定位器的图像集合姿态

2. Authors: Eric Brachmann, Jamie Wynn, Shuai Chen, Tommaso Cavallari, Áron Monszpart, Daniyar Turmukhambetov, Victor Adrian Prisacariu

3. Affiliation: Niantic（Niantic）

4. Keywords: Scene Coordinate Reconstruction, Incremental Learning, Visual Relocalization, Neural Scene Representations

5. Urls: [Paper](http://arxiv.org/abs/2404.14351v1), [Github](https://nianticlabs.github.io/acezero/)

6. Summary: 

   - (1): 该论文研究背景为图像集合中的相机参数估计。

   - (2): 过去的方法主要是基于特征匹配的结构运动，但存在诸如需要粗略初始姿态、姿态分布的先验知识或大量图像等问题。该论文提出的方法受增量结构运动和另一个计算机视觉任务，即视觉重定位的关系启发。过去的学习方法存在显著限制，如需要粗略初始姿态、姿态分布的先验知识或大量图像。但这些限制并非使用基于学习的场景表示的固有结果。

   - (3): 该论文提出的研究方法是基于增量学习的视觉重定位，允许对未被定位的图像进行姿态估计，并通过迭代应用和优化来建立隐式的神经场景表示。

   - (4): 该方法在无姿态先验和顺序输入的情况下，有效地优化数千幅图像，并实现与基于特征匹配的结构运动相媲美的相机姿态估计精度，通过新颖视图合成进行验证。

   - (5): 该论文的贡献在于提出了一种不需要姿态先验、能够处理大规模图像集合的图像重定位方法，为建立隐式的神经场景表示提供了新思路。

## Heterogeneous Face Recognition Using Domain Invariant Units
- **Url**: http://arxiv.org/abs/2404.14343v1
- **Authors**: ['Anjith George', 'Sebastien Marcel']
- **Abstrat**: Heterogeneous Face Recognition (HFR) aims to expand the applicability of Face Recognition (FR) systems to challenging scenarios, enabling the matching of face images across different domains, such as matching thermal images to visible spectra. However, the development of HFR systems is challenging because of the significant domain gap between modalities and the lack of availability of large-scale paired multi-channel data. In this work, we leverage a pretrained face recognition model as a teacher network to learn domaininvariant network layers called Domain-Invariant Units (DIU) to reduce the domain gap. The proposed DIU can be trained effectively even with a limited amount of paired training data, in a contrastive distillation framework. This proposed approach has the potential to enhance pretrained models, making them more adaptable to a wider range of variations in data. We extensively evaluate our approach on multiple challenging benchmarks, demonstrating superior performance compared to state-of-the-art methods.

1. Title: Heterogeneous Face Recognition Using Domain Invariant Units (使用域不变单元进行异构人脸识别)

2. Authors: Anjith George and Sébastien Marcel

3. Affiliation: Idiap Research Institute, Martigny, Switzerland

4. Keywords: Face Recognition, Heterogeneous Face Recognition, Domain Invariant Units, Biometrics

5. Urls: [Paper](http://arxiv.org/abs/2404.14343v1), [Github](None)

6. Summary: 

   - (1): 这篇文章的研究背景是人脸识别系统在处理不同域之间的人脸图像匹配时面临的挑战。

   - (2): 过去的方法包括基于特征的方法（如DoG和SIFT）和共享空间投影方法，以及最近的基于GAN的合成方法和Domain-Specific Units（DSU）等。这些方法的问题在于，它们要么需要大量数据训练，要么引入了计算成本较高的图像合成，限制了实际应用性。这些方法的动机合理，但在跨域匹配上仍有待改进。

   - (3): 本文提出了一种新的方法，即在教师-学生蒸馏学习框架中微调预训练的人脸识别网络组件，以学习域不变的特征。该方法旨在提高预训练模型对数据变化的适应性，使用了对比蒸馏框架进行有效训练，并引入了域不变单元（DIU）来减小域之间的差距。

   - (4): 该方法在多个标准基准测试中进行了广泛评估，表现优异，相比最先进的方法有了显著提升，支持其在解决异构人脸识别问题上的目标。

   - (5): 本文的主要贡献包括：在教师-学生蒸馏框架中阐明了异构人脸识别问题，并提出了一种能够通过少量配对数据改善预训练模型的方法；通过在多个标准基准上进行有效评估，证明了该方法的有效性；并承诺公开协议和源代码以促进进一步研究和应用。

## From Modalities to Styles: Rethinking the Domain Gap in Heterogeneous Face Recognition
- **Url**: http://arxiv.org/abs/2404.14247v1
- **Authors**: ['Anjith George', 'Sebastien Marcel']
- **Abstrat**: Heterogeneous Face Recognition (HFR) focuses on matching faces from different domains, for instance, thermal to visible images, making Face Recognition (FR) systems more versatile for challenging scenarios. However, the domain gap between these domains and the limited large-scale datasets in the target HFR modalities make it challenging to develop robust HFR models from scratch. In our work, we view different modalities as distinct styles and propose a method to modulate feature maps of the target modality to address the domain gap. We present a new Conditional Adaptive Instance Modulation (CAIM ) module that seamlessly fits into existing FR networks, turning them into HFR-ready systems. The CAIM block modulates intermediate feature maps, efficiently adapting to the style of the source modality and bridging the domain gap. Our method enables end-to-end training using a small set of paired samples. We extensively evaluate the proposed approach on various challenging HFR benchmarks, showing that it outperforms state-of-the-art methods. The source code and protocols for reproducing the findings will be made publicly available

1. Title: 从模态到风格：重新思考异构人脸识别中的领域差距

2. Authors: Anjith George, S´ebastien Marcel

3. Affiliation: Idiap Research Institute, Martigny, Switzerland

4. Keywords: Face Recognition, Heterogeneous Face Recognition, Style transfer, Instance Normalization, Biometrics

5. Urls: [Paper](http://arxiv.org/abs/2404.14247v1), Github: None

6. Summary:

   - (1): 这篇文章的研究背景是异构人脸识别领域。
 
   - (2): 过去的方法主要集中在在同质领域内进行人脸识别，但在跨领域匹配方面存在困难，主要是由于领域差距造成的。该方法的动机合理，因为跨领域匹配对于应对挑战性场景是至关重要的。
 
   - (3): 本文提出了一种新的方法来调节目标模态的特征图，以解决领域差距问题。作者引入了一种新的条件自适应实例调制（CAIM）模块，有效地调整中间特征图，适应源模态的风格，并弥合领域差距。这种方法使得可以使用一小组配对样本进行端到端的训练。
  
   - (4): 该方法在各种具有挑战性的异构人脸识别基准上进行了广泛评估，表明优于现有方法。性能支持了他们的目标。
  
   - (5): 本文的贡献在于提出了一种新的方法来解决异构人脸识别中的领域差距问题，并且在实验中表现出了优越的性能。
